### DISCUSSION TITLE
ValueError with informant.py

### DISCUSSION THREAD

**Discussion by NFernandes86** [TheBrownLab/PhyloFisher#113] [General]
Hello, 

When I run informant.py, it returns the following error: 

db_taxa_dict = tools.parse_metadata(args.metadata)
  File "~/miniconda3/envs/fisher/lib/python3.7/site-packages/phylofisher/tools.py", line 64, in parse_metadata
    org, long_name, group, subtax, _, _ = line.split('\t')
ValueError: not enough values to unpack (expected 6, got 5)

I am stuck at this stage, following the "Detailed example workflow". Can someone help me with this?
I am using Ubuntu 22.4 LTS

**Comment by atice** [comment]
Hello,

We have a PhyloFisher development team meeting tomorrow at noon. We will add your issue as a talking point. In the mean time, could you provide us with your input_metadata.tsv file so we can best trouble shoot your issue? Could you also let me know which version of PhyloFisher you are running currently.

Alex

Alexander K. Tice, Ph.D.
Assistant Professor
Department of Biological Sciences
Texas Tech University
Office: 313
Lab: 308 & 313A
Phone: (806) 834-6444
________________________________
From: NFernandes86 ***@***.***>
Sent: Tuesday, January 16, 2024 5:22 AM
To: TheBrownLab/PhyloFisher ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TheBrownLab/PhyloFisher] ValueError with informant.py (Discussion #113)

This email originated outside TTU. Please exercise caution<https://askit.ttu.edu/phishing>!


Hello,

When I run informant.py, it returns the following error:

db_taxa_dict = tools.parse_metadata(args.metadata)
File "~/miniconda3/envs/fisher/lib/python3.7/site-packages/phylofisher/tools.py", line 64, in parse_metadata
org, long_name, group, subtax, _, _ = line.split('\t')
ValueError: not enough values to unpack (expected 6, got 5)

I am stuck at this stage, following the "Detailed example workflow". Can someone help me with this?
I am using Ubuntu 22.4 LTS

**Reply by NFernandes86** [reply]
Thank you for your quick reply. Im running with the latest version 1.2.13. The input_metadata.tsv file is attached.
[input_metadata.txt](https://github.com/TheBrownLab/PhyloFisher/files/13951540/input_metadata.txt)

**Comment by atice** [comment]
Hi @NFernandes86 ,

Can you send me your metadata.tsv as well. Thanks.

Alex

**Reply by NFernandes86** [reply]
Hi Alexander,
It's the original you provided. I didn't modify it.
[metadata.txt](https://github.com/TheBrownLab/PhyloFisher/files/13975861/metadata.txt)

**Comment by atice** [comment]
Interesting. Somehow the Source column for the ciliates in the dataset was blank and the information that should have been in it was moved into the Data Type column. Try replacing the file with this one and running informant.py

[metadata.txt](https://github.com/TheBrownLab/PhyloFisher/files/13977677/metadata.txt)

**Comment by NFernandes86** [comment]
Now it's working :) thank you!
